homeassistant/components/http/auth.py

```diff
@@ -190,6 +190,11 @@ def setup_auth(hass, app):
 
         elif (trusted_networks and
               await async_validate_trusted_networks(request)):
+            _LOGGER.warning(
+                'Using trusted networks to bypass authentication is going to'
+                ' deprecate after 0.95 release. You need to use a bearer token'
```

Comment:
Do you mean removed? If so, it is deprecated now.

Reply:
Yes, removed